crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -44,6 +44,7 @@ CREATE TABLE notes (
     committed_at             INTEGER NOT NULL, -- Block number when the note was committed
     batch_index              INTEGER NOT NULL, -- Index of batch in block, starting from 0
     note_index               INTEGER NOT NULL, -- Index of note in batch, starting from 0
+    transaction_id           BLOB    NOT NULL, -- ID of the transaction which emitted this note
```

Comment:
Technically with this addition we can remove `committed_at` and `sender` since these are tracked as part of the transaction data. This might make other queries more painful though so I'm unsure if we should bother at this stage.

Maybe add an issue to look into this at a later stage.